src/widgets/calendar/lv_calendar_lunar.h

```diff
@@ -0,0 +1,66 @@
+/**
+ * @file lv_calendar_lunar.h
+ *
+ */
+
+#ifndef LV_CALENDAR_LUNAR_H
+#define LV_CALENDAR_LUNAR_H
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/*********************
+ *      INCLUDES
+ *********************/
+#include "../../core/lv_obj.h"
+#if LV_USE_CALENDAR && LV_USE_CALENDAR_LUNAR
+
+/*********************
+ *      DEFINES
+ *********************/
+
+/**********************
+ *      TYPEDEFS
+ **********************/
+
+typedef struct {
+    const char festival_name[20];
+    uint8_t month;
+    uint8_t day;
+} lv_calendar_festival;
+
+typedef struct {
+    lv_calendar_date_t today;
+    bool leep_month;
+} lv_calendar_lunar;
```

Comment:
Move these into the .c file if they aren't part of the exposed API and add `_t` to the name.

Reply:
ok